spec/integration/default_spec.rb

```diff
@@ -70,6 +70,16 @@
   it { should_not be_file }
 end
 
+describe file('/tmp/http_request.html') do
+  it { should be_file }
+  its(:content) { should match(/Itamae/) }
```

Comment:
This looks assuming `http://itamae.kitchen` always return a body including `Itamae` but it can change.